makefiles/bettercap.mk

```
@@ -0,0 +1,54 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS    += bettercap
+BETTERCAP_VERSION  := 2.32.0
+DEB_BETTERCAP_V    ?= $(BETTERCAP_VERSION)
+
+bettercap-setup: setup
+	$(call GITHUB_ARCHIVE,bettercap,bettercap,$(BETTERCAP_VERSION),v$(BETTERCAP_VERSION),bettercap)
+	$(call EXTRACT_TAR,bettercap-$(BETTERCAP_VERSION).tar.gz,bettercap-$(BETTERCAP_VERSION),bettercap)
```

The last argument in `GITHUB_ARCHIVE` is not necessary

```suggestion
	$(call GITHUB_ARCHIVE,bettercap,bettercap,$(BETTERCAP_VERSION),v$(BETTERCAP_VERSION))
	$(call EXTRACT_TAR,bettercap-$(BETTERCAP_VERSION).tar.gz,bettercap-$(BETTERCAP_VERSION),bettercap)
```